### PR TITLE
Create orangethewell.json

### PR DESCRIPTION
**Pull request by orangethewell**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] You're not using Vercel or Netlify.  
- [ ] The CNAME record doesn't contain `https://` or `/`.  
- [ ] There is sufficient information at the `owner` field.  

## Website Link/Preview

http://85.31.231.4:5080/

### CODE REVIEW

**Review by triach-rold — COMMENTED** [review]
Pretty sure the email address has a typo in it.

**Review by triach-rold — CHANGES_REQUESTED** [review]
Website is non empty, but reachable. Email address (probably?) has a typo in it because of the first character being "<" and the URL is only reachable at 85.31.231.4:5080/ and NOT at 85.31.231.4 on my system.

**Comment by orangethewell** [comment]
> Website is non empty, but reachable. Email address (probably?) has a typo in it because of the first character being "<" and the URL is only reachable at 85.31.231.4:5080/ and NOT at 85.31.231.4 on my system.

The port is because I plan to add nginx to handle a reverse proxy to it and the server is used for more than one website. And the email... I didn't understand, the email should be between "<" and ">"?

**Comment by orangethewell** [comment]
@triach-rold should I move the port of my website temporary to port 80 to be able to be approved?

File: domains/orangethewell.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "owner": {
+    "username": "orangethewell",
+    "email": "orangethewell@gmail.com"
+  },
+
+  "record": {
+    "A": "85.31.231.4"
```
Comment (CuteDog5695) [inline domains/orangethewell.json:8]: IP address needs to be in an array

**Review by iostpa — CHANGES_REQUESTED** [review]

**Comment by CuteDog5695** [comment]
> @triach-rold should I move the port of my website temporary to port 80 to be able to be approved?

doesnt matter if it has ports or not the website, it will be merged (tho file is invalid so fix that first)

**Comment by triach-rold** [comment]
> @triach-rold should I move the port of my website temporary to port 80 to be able to be approved?

Nope, I don't think you need to do that

**Comment by CuteDog5695** [comment]
no he doesn't, as long as the website is hosted on the IP it will be able to connect his domain after being merged

**Review by iostpa — APPROVED** [review]
seems good to me now

**Pull request merged**